yb-voyager/cmd/common.go

```diff
@@ -308,6 +318,73 @@ func retrieveMigrationUUID(exportDir string) error {
 	return nil
 }
 
+func getMetaDBPath(exportDir string) string {
```

Comment:
How about creating a new file `metadata-db.go` for all functions related to the meta db?